lua/entities/gmod_wire_textentry.lua

```diff
@@ -187,7 +197,7 @@ function ENT:OnTextEntered(text)
 	WireLib.TriggerOutput( self, "Entered", 1 )
 	WireLib.TriggerOutput( self, "Entered", 0 )
 
-	local timername = "wire_textentry_" .. self:EntIndex()
+	local timername = self:GetHoldTimerName()
 	timer.Remove( timername )
```

Comment:
This can be `self:RemoveHoldTimer()` right?

Reply:
Yes, but then I'll execute `self:GetHoldTimerName()` twice